### ISSUE TITLE
Getting onnxruntime error when running the inference session

### ISSUE THREAD

**Issue by ac4922** [onnx/models#164]
I am trying to run the inference with the below code :--

sess = rt.InferenceSession(model_onnx_file)
x = cv2.imread('Objectives.png')
x= cv2.resize(x, (224,224))
x = x[numpy.newaxis, :, :, :]
x= x.astype(numpy.float32)
print(x.shape)
predicted_value = model_inference_session.run(None, {input_name: x})[0]

This is what I am trying to do after downloading the onnx file from the master version. 

I am getting the below error :--

RuntimeError: Method run failed due to: [ONNXRuntimeError] : 1 : GENERAL ERROR : Non-zero status code returned while running Node:  Status Message: Input channels C is not equal to kernel channels * group. C: 224 kernel channels: 3 group: 1

Could you please suggest how to resolve this?

**Comment by prasanthpul** [comment]
which model are you using?

**Comment by ac4922** [comment]
VGG19

**Comment by ac4922** [comment]
> which model are you using?

VGG19

**Comment by pranavsharma** [comment]
The error message clearly says - Input channels C is not equal to kernel channels * group. C: 224 kernel channels: 3 group: 1. This can be resolved by fixing the shape of your input. @hariharans29

**Comment by hariharans29** [comment]
Hi @ac4922,

* When you do `print(x.shape)`, is your input shape [1, 224, 224, 3] ? According to the [documentation] 
   (https://github.com/onnx/models/tree/master/vgg19), the shape of the input tensor needs to be [1,3, 
   224, 224].

* Since channels are in the last dimension, you need a transpose of some sort like this - np.transpose(img_data, [2, 0, 1]) *before* you add the batch dimension.

In short please make sure, you input tensor is of the shape [1, 3, 224, 224] before running the model as the documentation for the model says.

**Comment by prasanthpul** [comment]
Thanks @hariharans29 

@ac4922 i'll close the issue here since this does not seem like an issue with the ONNX model. suggest you open an issue for ONNX Runtime at https://github.com/microsoft/onnxruntime if you would like to see improvements in the error message.

**Comment by ac4922** [comment]
> Hi @ac4922,
> 
> * When you do `print(x.shape)`, is your input shape [1, 224, 224, 3] ? According to the [documentation](https://github.com/onnx/models/tree/master/vgg19), the shape of the input tensor needs to be [1,3, 224, 224].
> * Since channels are in the last dimension, you need a transpose of some sort like this - np.transpose(img_data, [2, 0, 1]) _before_ you add the batch dimension.
> 
> In short, please make sure your input tensor is of the shape [1, 3, 224, 224] before running the model as the documentation for the model says.

Thank you :) 👍

**Comment by ac4922** [comment]
> Thanks @hariharans29
> 
> @ac4922 i'll close the issue here since this does not seem like an issue with the ONNX model. suggest you open an issue for ONNX Runtime at https://github.com/microsoft/onnxruntime if you would like to see improvements in the error message.

Sure . Thank you :) 👍
